wit/deps/keyvalue-2024-05-03/world.wit

```diff
@@ -0,0 +1,26 @@
+package wasi:keyvalue@0.2.0-draft;
```

Comment:
Are we planning to remove `-draft`?

Reply:
Is there a non-draft WIT yet?  The HEAD in the wasi-keyvalue GitHub still has `draft` at time of writing, but if there's a non-draft world then yeah we should definitely use that instead.

https://github.com/WebAssembly/wasi-keyvalue/blob/219ea3612a53f1bf5b2d137551b22d0268fd3c58/wit/world.wit#L1

Reply:
I see. I missed that in the versioning of the upstream WIT. Thanks